duckduckgo/iOS
# Remove Light Theme experiment, enable it for everyone + final touches

Task/Issue URL: https://app.asana.com/0/856498667320406/943952882631812
Tech Design URL: N/A

Description:
We are removing the experiment and enabling theme switching for everyone, with Dark Theme as a default. 

Steps to test this PR:

1. App Upgrade (no settings change):

- Install develop version of the App, with 'mh' experiment running (default light theme).
- Now install this branch over the top.
- Ensure light theme selection was maintained.

2. App Upgrade (with settings change):

- Install develop version of the App, with 'mj' experiment running (default dark theme).
- In settings, enable the light theme.
- Now install this branch over the top.
- Ensure light theme selection was maintained.

3. New installation

- Install a fresh version of the app from this branch.
- Ensure that the dark theme is the default.
- Ensure the theme can be changed in the settings.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
